package.json

```diff
@@ -18,7 +23,17 @@
   },
   "homepage": "https://github.com/yan-fung/music-library-#readme",
   "devDependencies": {
+    "chai": "^4.3.7",
```

Comment:
Can't leave these comments in a better place:
(1) you have a .prettierrc.json file but you don't have prettier as a dependency in package.json
(2) you have eslint as a dependency in the package.json but no .eslintrc.json to configure it. 
I would recommend both of these for formatting and consistency